apps/chat/src/store/share/share.reducers.ts

```diff
@@ -199,6 +195,16 @@ export const shareSlice = createSlice({
     ) => {
       state.shareModalState = payload.modalState;
     },
+    setUnshareModel: (
+      state,
+      {
+        payload,
+      }: PayloadAction<{
+        entity?: DialAIEntityModel;
+      }>,
+    ) => {
+      state.unshareEntity = payload.entity;
+    },
```

Comment:
As I can see we use only `id, name, isShared, sharedWithMe` from `unshareEntity` i.e. we can use type `Omit<ShareEntity, 'folderId'>` instead of `DialAIEntityModel` and use consistent name. I would be more common and not model specific
```suggestion
    setUnshareEntity: (
      state,
      { payload }: PayloadAction<Omit<ShareEntity, 'folderId'> | undefined>,
    ) => {
      state.unshareEntity = payload;
    },
```